response.md

```diff
@@ -0,0 +1,86 @@
+
+Much of the blog post is dedicated to discussing country flags, and using them as the basis for a justification for rejecting all other flags.  Let's break down their points and explain why they are not a valid justification for rejecting all flags going forward.
+
+**There are over 200 flags, and many of them are among the least used emoji.**
+
+This point is true, but largely irrelevant.  Country flags are often a sign of national pride, so countries that have a higher population, or a population that is more on-line, or that have a better sports team for the Olympics or World Cup, will tend to have more flag use, because there are more people who are interested in using that flag.  It's not always a direct correlation (China and India have populations several times the population of the US, but their flags are used less.), but it is a general trend.
+
+Take, for example, Liechtenstein.  The Liechtenstein flag emoji consistently ranks very low on lists of emoji usage frequency.  That's understandable, because the population of Liechtenstein is around 40,000 people, less than the typical in-person attendance of an NFL game.  It is likely that the flag of Liechtenstein sees less use than the Seattle Seahawks "12th Man" flag.   But it's not just people who are physically at the game who might use that flag, it's the entire population of the Seattle sports market, which is roughly five million people.  About half the countries in the world have a population of less than five million people.  So is the "12th Man" flag more popular than half the national flags in the world?
+
+There is a clear structural reason here for why the existing flags as a category see low use, and it is disingenuous to extrapolate that all flags will also see such limited use.  A common estimate of the prevalence of asexual people would indicate that there are at least 79 million aces in the world.  That's more than the population of the UK and France, and just shy of the population of Germany, all of which have country flags that see above median use.  Some estimates claim that the number of bisexual people worldwide rivals the population of the United States, the third most populous country in the world, and holder of the most used flag emoji.  Going by these numbers, it is reasonable to conclude that pride flag emojis would out-perform the majority of the existing flag emojis.
```

Comment:
Minor: source? quick google search led me here https://williamsinstitute.law.ucla.edu/press/sm-asexuals-press-release/

Reply:
for bisexual count: https://www.ipsos.com/sites/default/files/ct/news/documents/2021-06/LGBT%20Pride%202021%20Global%20Survey%20Report_3.pdf